docker/Dockerfile-api

```diff
@@ -1,4 +1,4 @@
-FROM fedora:30
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
Please pin the parent image tag

Reply:
@lioramilbaum I'd rather not pin it since Red Hat will ensure compatibility and thus I'd rather just always be on the latest UBI8 to inherit security fixes on deployments.

Reply:
Unpinned image tag will prevent us from reproducing the exact environment running in production (for example). Using ephemeral tags is an anti pattern as it comes to reproducible environments.